bin/apostrophe

```diff
@@ -2,60 +2,67 @@
+  program.version(getVersion());
 
-// Get version of apostrophe-cli and apostrophe.
-function getVersion() {
-  var pkginfo = require('pkginfo')(module, 'version');
-  var cwd = process.cwd();
+  // Get version of apostrophe-cli and apostrophe.
+  function getVersion() {
+    require('pkginfo')(module, 'version');
+    var cwd = process.cwd();
 
-  var output = 'apostrophe-cli: v' + module.exports.version + '\n';
-  var aposPath = cwd + '/node_modules/apostrophe/';
+    var output = 'apostrophe-cli: v' + module.exports.version + '\n';
+    var aposPath = cwd + '/node_modules/apostrophe/';
 
-  // Append message for apostrophe.
-  if (fs.existsSync(aposPath)) {
-    var package = require(aposPath + 'package.json');
+    // Append message for apostrophe.
+    if (fs.existsSync(aposPath)) {
+      var aposPkg = require(aposPath + 'package.json');
 
-    output += 'apostrophe v' + package.version + ' is installed in this project.';
-  } else {
-    var request = require('sync-request');
-    var response = request('GET', 'https://raw.githubusercontent.com/apostrophecms/apostrophe-boilerplate/master/package.json');
+      output += 'apostrophe v' + aposPkg.version + ' is installed in this project.';
+    } else {
+      var request = require('sync-request');
+      var response = request('GET', 'https://raw.githubusercontent.com/apostrophecms/apostrophe-boilerplate/master/package.json');
 
-    if (response.statusCode == 200) {
-      var packageJSON = JSON.parse(response.body);
+      if (response.statusCode === 200) {
+        var packageJSON = JSON.parse(response.body);
 
-      output += 'apostrophe ' + packageJSON.dependencies.apostrophe + ' will be installed with a new project, according to the dependencies of apostrophe-boilerplate.';
-    } else {
-      var execSync = require('child_process').execSync;
-      var aposVersion = execSync('npm view apostrophe version').toString();
+        output += 'apostrophe ' + packageJSON.dependencies.apostrophe + ' will be installed with a new project, according to the dependencies of apostrophe-boilerplate.';
+      } else {
+        var execSync = require('child_process').execSync;
+        var aposVersion = execSync('npm view apostrophe version').toString();
 
-      output += 'apostrophe v' + aposVersion.trim() + ' (latest) will be installed with a new project, unless the boilerplate project\'s dependencies specify otherwise.';
+        output += 'apostrophe v' + aposVersion.trim() + ' (latest) will be installed with a new project, unless the boilerplate project\'s dependencies specify otherwise.';
+      }
     }
+
+    return output;
   }
 
-  return output;
-}
+  util.checkDependencies();
 
-util.checkDependencies();
+  require('../lib/commands/create')(program);
+  require('../lib/commands/create-widget')(program);
+  require('../lib/commands/create-piece')(program);
+  require('../lib/commands/create-module')(program);
+  var aposCmd = require('../lib/apostrophe')(program);
```

Comment:
Apostrophe 2 no longer runs on Node 6, so you may as well go const/let, but don't let it slow you down, this is great.

Reply:
Totally. I didn't want to clutter this PR up with all those diffs. Certainly a quick cleanup project for the future.